pycose/keys/okp.py

```diff
@@ -41,6 +43,77 @@ def from_dict(cls, cose_key: dict) -> 'OKPKey':
 
         return cls(crv=curve, x=x, d=d, optional_params=_optional_params, allow_unknown_key_attrs=True)
 
+    @classmethod
+    def from_cryptography_key(
+        cls,
+        ext_key: Union[
+            ed25519.Ed25519PrivateKey, ed25519.Ed25519PublicKey,
+            ed448.Ed448PrivateKey, ed448.Ed448PublicKey,
+            x25519.X25519PrivateKey, x25519.X25519PublicKey,
+            x448.X448PrivateKey, x448.X448PublicKey
+        ],
+        optional_params: Optional[dict] = None,
+    ) -> 'OKPKey':
+        """
+        Returns an initialized COSE Key object of type OKPKey.
+        :param ext_key: Python cryptography key.
+        :return: an initialized OKP key
+        """
+
+        curve = cls._curve_from_cryptography_key(ext_key)
+
+        if hasattr(ext_key, 'private_bytes'):
+            priv_bytes = ext_key.private_bytes(
+                encoding=Encoding.Raw,
+                format=PrivateFormat.Raw,
+                encryption_algorithm=NoEncryption(),
+            )
+            pub_bytes = ext_key.public_key().public_bytes(
+                encoding=Encoding.Raw, format=PublicFormat.Raw
+            )
```

Comment:
Obviously you can't use it since it's not even released yet, but I just noticed this: https://github.com/pyca/cryptography/commit/db6905d69c4a1330570b00210e057bd3ecfbe7d3

Maybe something to keep in mind for in a few months time.